### DISCUSSION TITLE
What implications there are when using no indexer? (indexer=null)

### DISCUSSION THREAD

**Discussion by Yukigaru** [tendermint/tendermint#10102] [General]
I'm experiencing CPU/IO spikes every 7-8 sec in my dydx node and go perf shows they are partly related to a DB functions. My questions are:
- can I disable indexer and have the node work without it?
- what are implications? 
Thank you!

**Comment by andynog** [comment]
Hi @Yukigaru, you can disable the indexer by setting it to 'null' in the configuration file. If you disable the indexer the implication is that you won't be able to query the '/tx_search' and '/block_search' endpoints.

https://docs.cometbft.com/v0.38/app-dev/indexing-transactions

**Reply by Yukigaru** [reply]
thank you!

**Comment by andynog** [comment]
By the way, this repository is not maintained anymore, please use cometbft repo in the future, thanks!
